### ISSUE TITLE
Need usage example in the readme

### ISSUE THREAD

**Issue by junaid7lone** [JedWatson/react-scroll-spy#1]
Currently there is no example on how to use this library, for a novice like me its quite difficult to understand.
Thanks

**Comment by Kikketer** [comment]
Agreed, I dove through the source code (which is small and delightful) but still can't figure out how to tag elements to switch from "notInView" to "inView".
I'd like to use this library but unfortunately I have things to do, so I'll be moving on.

**Comment by ninahaack** [comment]
I couldn't use this library because there isn't any example
